packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -874,10 +949,11 @@ export class StatsAnalyzer extends EventsScope {
   private processOutboundRTPResult(result: any, mediaType: any) {
     const sendrecvType = STATS.SEND_DIRECTION;
 
+    this.processTrackResult(result, mediaType, sendrecvType);
     if (result.bytesSent) {
       let kilobytes = 0;
 
-      if (result.frameWidth && result.frameHeight) {
+      if (result.frameWidth && result.frameHeight && !mediaType.includes('audio')) {
```

Comment:
why do we need this now? the code has been working fine so far without it, right? also if you add it here, then why not also for framesEncoded and keyFramesEncoded a few lines below (and more of it further below)

Reply:
Removed this.